src/main/java/org/ibp/api/java/impl/middleware/ontology/validator/VariableValidator.java

```diff
@@ -152,6 +162,61 @@ private boolean nameValidationProcessor(final VariableDetails variable, final Er
 		return errors.getErrorCount() == initialCount;
 	}
 
+	private boolean aliasValidationProcessor(final VariableDetails variable, final Errors errors) {
+
+		if (StringUtils.isBlank(variable.getAlias())) {
+			return true;
+		}
+
+		if(variable.getAlias().equalsIgnoreCase(variable.getName())){
+			this.addCustomError(errors, "alias", BaseValidator.NAME_AND_ALIAS_ARE_EQUALS, null);
+		}
+
+		final Integer initialCount = errors.getErrorCount();
```

Comment:
Is errorsCount <> 0 when calling this function?

Reply:
done.